config/routes.rb

```diff
@@ -1,3 +1,12 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  get "/guests/:id", to: "guests#show" 
+
+  post "/guest_rooms/:guest_id", to: "guest_rooms#create"
```

Comment:
I'm ok with this route, but overall if a route is creating something, it won't have an ID in the path; if we're creating a relationship from Guest to Room, then both could be contained within the form data sent along in the request body.